src/refiners/foundationals/dinov2/vit.py

```diff
@@ -0,0 +1,227 @@
+from typing import Any
+
+import torch
+from torch import Tensor
+
+import refiners.fluxion.layers as fl
+from refiners.fluxion.layers.activations import Activation
+
+
+class CLSToken(fl.Parameter):
```

Comment:
nit: it is named `ClassEncoder` in https://github.com/finegrain-ai/refiners/blob/0dc3a17fbf442f5b7f38813d77752dec08bc26db/src/refiners/foundationals/clip/image_encoder.py#L6

You could use this naming here too for consistency (note that the implementation slight vary - Chain vs. Parameter subclass)